styles.css

```diff
@@ -0,0 +1,669 @@
+@font-face {
+  font-family: cocogoose;
+  src: url(fonts/Cocogoose-font.otf);
+}
+
+html,
+body {
```

Comment:
* Kindly fix the width of the page currently the width overflows please adjust it to fit the screen 
![image](https://user-images.githubusercontent.com/52497246/133898444-4bd2de66-8b40-402a-be3a-ca3b55d650f2.png)

Reply:
done